app/frontend/src/locales/da/translation.json

```diff
@@ -79,7 +79,8 @@
         "retrieveCount": "Hent dette antal søgeresultater:",
         "includeCategory": "Inkludér kategori",
         "includeCategoryOptions": {
-            "all": "Alle"
+            "all": "Alle",
+            "none": "Ingen"
```

Comment:
@EMjetrot Can you review this string- "Ingen" for "None" , as in "no categories"?